pages/wishlist.md

```diff
@@ -6,28 +6,28 @@ permalink: /wishlist/
 
 As many excellent apps you will find here, as many people probably miss their personal favorite. This is just a little collection of elementary applications that are either waiting to be revived, only live in the form of mockups or even only in the heads of wishful thinkers. 
 
-If you are a developer please squash some [elementary bugs](http://bugs.launchpad.net/elementary) or help out [other developers]({{ site.baseurl }}/apps/) first. Or do you own thing: Any item on this list is a good place to start your own elementary project. 
+If you are a developer please squash some [elementary bugs](http://bugs.launchpad.net/elementary) or help out [other developers]({{ site.baseurl }}/apps/) first. Or do you own thing: Any item on this list is a good place to start your own elementary project.
 
 If there is another app for elementary you are missing desperately, feel free to [file an issue at GitHub](https://github.com/quassy/elementary-apps/issues/new) to get it added to the list (or develop it yourself and tell us about it). 
 
 Name | Generic | Info
 ---|---|---
-[AppCenter](https://launchpad.net/appcenter) (in-development) | Software Store | Official elementary app. Only included because people often ask: This in development, but it is not expected to be finished before elementary 0.4 Loki. 
```

Comment:
While not officially announced Loki was floating around for some time and is being used on Launchpad by elementary itself. Even if it does not become the release name, it certainly is the current development codename, similar to Longhorn or Treshold.